setup.cfg

```diff
@@ -42,6 +42,7 @@ install_requires =
     requests
     requests_cache
     rich
+    python-slugify
```

Comment:
A small dependency for converting any title to a latex-compatible slug version of it